canonical/lxd-ui
# Use shorter urls, without /detail/ in path

## Done

- shortened urls, so detail pages use singular forms, before: /ui/ui/project/default/instances/detail/holy-quagga now: ui/project/default/instance/holy-quagga. Same for all other pages with list and detail pages

Fixes [list issues/bugs if needed]

## QA

1. Run the LXD-UI:
    - On the demo server via the link posted by @webteam-app below. This is only available for PRs created by collaborators of the repo. Ask @mas-who or @edlerd for access.
    - With a local copy of this branch, run as described [here](../HACKING.md#setting-up-for-development).
2. Perform the following QA steps:
    - spot check links in the UI
